internal/collector/utils.go

```diff
@@ -0,0 +1,39 @@
+package collector
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/sirupsen/logrus"
+)
+
+func keepalivedVersion() *version.Version {
+	cmd := exec.Command("bash", "-c", "keepalived -v")
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	if err != nil {
+		logrus.WithFields(logrus.Fields{"stderr": stderr.String(), "stdout": stdout.String()}).WithError(err).Fatal("Error getting keepalived version")
```

Comment:
Log an error and fallback to the latest version we know.

Reply:
This goes to all the error handlings in this function.

Reply:
Can you explain more why? :thinking:

Reply:
I don't think not being able to detect the keepalived's version should be a fatal error. If it's possible to do its job without knowing the version and assuming it's the latest one (As we were doing before this change) we should do it.

Reply:
Agree :+1: